docs/users-guide/models.md

```diff
@@ -106,6 +106,18 @@ Just like with saved questions, you can click on the model name with the down ar
 
 Just like with a question, admins can verify models. Verifying a model will give it a check mark to let others know an admin vetted the model. If anyone makes any changes to the model, the check mark will disappear. An admin will have to verify the question again to restore the check mark.
 
+## Model caching
+
+You can enable caching for models so that loading questions based on those models is significantly faster. Instead of running the query that creates the model fresh each time, Metabase will store the results in a table, and simply fetch the saved results table to quickly return the model. Caching models will make questions built on models run much faster. We recommend scheduling the cache to refresh on a schedule that makes sense with how often your source tables update with new data.
```

Comment:
```suggestion
You can enable caching for models so that loading questions based on those models is significantly faster. Instead of running the query that creates the model fresh each time, Metabase will store the results in a table, and simply fetch the saved results table to quickly return the model. Caching models will make questions built on models run much faster. We recommend scheduling the cache to refresh on a frequency that makes sense with how often your source tables update with new data.
```

"frequency" to match the description of the cache settings below